packages/extension/webpack.config.js

```diff
@@ -76,11 +76,6 @@ module.exports = {
 
   resolve: {
     extensions: ['.svg', '.ts', '.tsx', '.js', '.json'],
-    alias: {
-      'webextension-polyfill-ts': path.resolve(
-        path.join(__dirname, 'node_modules', 'webextension-polyfill-ts'),
-      ),
-    },
```

Comment:
@rebelchris not sure what was the reason behind this alias in the past, but just wanted to tag you to let you know.

Reply:
Yeah I was just looking at that, but don't know.
Might have been legacy issue for lerna or something.

If everything works let's leave it out.

Reply:
Yep. Everything was working including the companion. So we should be safe :shipit: